src/python/platforms/fuchsia/device.py

```diff
@@ -31,7 +31,7 @@
 from system import shell
 
 
-def qemu_setup():
+def setup_qemu_values(initial_setup=True):
```

Comment:
nit: it seems like the common case is initial_setup=False, so maybe use that as the default?

Reply:
actually, looking at this a bit more, a better way would be to factor out the initial setup into its own function, which we call from build_setup.

Reply:
(i didn't do this -- please do this refactor in another PR if you agree)